src/value.cc

```diff
@@ -312,6 +312,46 @@ Local<Value> ArrayToV8 (GITypeInfo *type_info, void* data, long length) {
     return array;
 }
 
+long GIArgumentToLength(GITypeInfo *type_info, GIArgument *arg, bool is_pointer) {
```

Comment:
Hopefully, this does the right thing. I assume, we are stepping on undocumented GIR ground here again?
Nevertheless this seems at least better than the original line handling it.

Reply:
Everywhere is undocumented GIR ground. Everywhere.

But yes, this is more correct because it will work on machines with different endianness. Who knows, maybe someone will try to run node-gtk on a PowerPC.